### DISCUSSION TITLE
Where to change motor PWM switching frequency

### DISCUSSION THREAD

**Discussion by TCIII** [NVIDIA-AI-IOT/jetbot#463] [Q&A]
Presently Adafruit is using a PWM switching frequency of 100Hz which is fine for servos, but causes an annoying whine from my Jetbot's DC motors.
Therefore I would like to know where in the Adafruit motor driver packages can I change the PWM switching frequency from 50Hz to around 2k-2.5kHz?

Apparently the Jetbot 0.4.3 image no longer stores the Adafruit_MotorHAT packages here: “/usr/local/lib/python3.6/dist-packages/Adafruit_MotorHAT-1.4.0-py3.6.egg/Adafruit_MotorHAT/Adafruit_MotorHAT_Motors.py” as the “dist-packages” directory is empty now.

**Comment by TCIII** [comment]
I received the following from the Adafruit Customer Service:

That library is deprecated and has been archived:
https://github.com/adafruit/Adafruit-Mo ... on-Library

The new library is here:
https://github.com/adafruit/Adafruit_Ci ... n_MotorKit

Install info in guide here:
https://learn.adafruit.com/adafruit-dc- ... g-software

**Reply by irvinmarceloc** [reply]
Thanks for your answer, I am search this information. Great.
